velox/type/Type.h

```diff
@@ -761,6 +781,10 @@ class UnknownType : public TypeBase<TypeKind::UNKNOWN> {
     return 0;
   }
 
+  bool isOrderable() const override {
+    return false;
```

Comment:
strangely enough, UNKNOWN type in Presto is orderable

Reply:
Fixed